jena-fuseki2/jena-fuseki-main/src/test/java/org/apache/jena/fuseki/main/TestFusekiMainCmd.java

```diff
@@ -102,4 +103,11 @@ private void server(String... cmdline) {
             assertNotEquals(0, x.readAllBytes().length);
         }
     }
+
+    @Test public void tasks_01() {
+        server("--mem", "--tasks", "/ds");
+        String x = HttpOp.execHttpGetString(serverURL+"/$/tasks");
+        assertNotNull(x);
+        assertNotEquals(0, x.length());
```

Comment:
`JSON.parse` throws an error when calling it on `x` here because the root type of the JSON returned is a list, not an object. I opted to not update the parser to handle lists because it seemed silly to update it for this one test.

Reply:
`JSON.parseAny` will read a list or array (or a atomic JSON type).

`jena-fuseki-weabapp TestAdmin` has some tests of tasks.

Reply:
fixed.